### PR TITLE
Add data load instructions to README

### PR DESCRIPTION
**Pull request by mmcfarland**
Keeps in the instructions up to date with the latest setupdb.sh script usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +43,22 @@ After pulling in new commits, you may need to run the following two commands:
 
 ```bash
 $ ./scripts/manage.sh migrate
-$ ./scripts/manage.sh reload_dev_data
 $ ./scripts/bundle.sh
 ```
 
+To load or reload boundary data, from an `app` server, run(`scripts` is not mounted by default to the VM, you may need to copy the file over):
```
Comment (caseypt) [inline README.md]: Missing space between "... run (`scripts` ..."

File: README.md [hunk 2]
```diff
@@ -43,10 +43,22 @@ After pulling in new commits, you may need to run the following two commands:
...
+```bash
+$ ./scripts/setupdb.sh -b
+```
+
+The same script can be used to load the stream network data:
+```bash
+$ ./scripts/setupdb.sh -s
+```
+
+Note that if you receive out of memory errors while loading the data, you may want to increase the RAM on your `services` VM.
```
Comment (caseypt) [inline README.md]: 1512 mb appears to be working for me.

**Comment by caseypt** [comment]
+1

**Comment by mmcfarland** [comment]
Updated with whitespace fix and mention sufficient target memory amount.

**Pull request merged**